app/models/subject_queue.rb

```diff
@@ -79,6 +79,10 @@ def below_minimum?
   end
 
   def next_subjects(limit=10)
-    set_member_subject_ids[0..limit-1]
+    if user_id
+      set_member_subject_ids[0..limit-1]
```

Comment:
Won't all logged in users get the same sort order based on the `create_for_user` copying the `logged_out_queue` on queue creation? So if there is a spike in traffic there might be an uneven distribution of classifications (dependant on the size of the spike).

Perhaps introducing some randomised sorting into the `set_member_subject_ids` when creating the queue would alleviate this and even out users effort.